apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -62,6 +62,16 @@ function wpcom_should_limit_global_styles( $blog_id = 0 ) {
 		return false;
 	}
 
+	// Do not limit Global Styles when live previewing a Woo theme without a Business plan or higher.
```

Comment:
Thanks! I have implemented that logic but struggled while testing it because `wpcom_site_has_feature( WPCOM_Features::GLOBAL_STYLES, $blog_id )` always returns `false` when a sandboxed site has a premium plan or higher.

I've asked T-Rex about that in p1701937616146419-slack-C04DZ8M0GHW